### ISSUE TITLE
!! Better use Icon Shortcode Plugin !!

### ISSUE THREAD

**Issue by Kettenotter** [visini/obsidian-icons-plugin#27]
This plugin isn't updated anymore. But still further at the top than https://github.com/aidenlx/obsidian-icon-shortcodes in the plugin list.  
The other one supports:
-  mobile devices!
-  custom icons
- icon visible in edit mode
- emojis

Just leaving this here so people don't need to replace all their icons like me. 
Also it has the same icon packs which you can download under the plugin settings.

**Comment by ITMighty** [comment]
Unfortunately, that plugin has not been updated for just about 2-years and has issues that do not look like they'll be addressed by the author.

**Comment by whoszatarash** [comment]
Hey there @Kettenotter!

I sadly had to make a temporal switch to mobile and realized how important this plugin was for my Vault. While on PC it worked flawlessly, having a `code` on mobile makes it look like something's missing, as if I left things unfinished, and turns out its because icons can't be rendered on mobile.

I was resigned to accept it as it was until I searched around here a month or two ago and found out — much to my regret — that this plugin was abandoned years ago.

I found this post and decided to gave it a try but couldn't get it to work; in all honesty, the comment above mine discouraged me a bit. It wasn't until now that I definitely got tired of the situation and gave it a try again, and after a couple of minutes voilà! SVG renders on my phone! They weren't impossible at all!

Took my time to posting this to say huge thanks for sharing, if it wasn't for you I wouldn't have discovered such plugin. Although ITMighty might (npi) be correct due to this new plugin ALSO being abandoned (seriously, what's the deal behind icon plugins that seem to share fates?), my advice to others would be to give it a try by yourself. Really appreciate it

***Some packages don't even download and stumbled upon few buggy situations*** but still *managed to add my owns*, and that's enough for me.

TL;DR: Thank you so much for sharing! Worked like a charm. May your pillow be always the way you want it to be :D

Have a nice week!
_Best regards, **Z**_
